### ISSUE TITLE
goog.abstractMethod vs @abstract

### ISSUE THREAD

**Issue by fredj** [google/closure-compiler#1988]
Since 08606a76, the compiler warn if abstract methods appear in concrete classes:
```js
/**
 * @abstract
 * @constructor
 */
C = function() {
  this.foo = -1;
};

/**
 * @abstract
 */
C.prototype.add = function() {};
```
But it is not the case when `goog.abstractMethod` is used:
```js
/**
 * @constructor
 */
C = function() {
  this.foo = -1;
};

/**
 *
 */
C.prototype.add = goog.abstractMethod;
```
(and `goog.abstractMethod` doesn't require the constructor to be abstract.)

**Comment by Dominator008** [comment]
I haven't add this warning to `goog.abstractMethod` due to the risk of breaking many internal projects at Google.
